liiatools_pipeline/ops/common_la.py

```diff
@@ -102,6 +102,8 @@ def process_files(
         return
     log.info(f"{la_name} is signed for PAN data processing.")
 
+    output_config = pipeline_config(config)
```

Comment:
Any reason why you've created this variable instead of just calling pipeline_config(config) where necessary? Just as I'm not sure this is the way dagster assets are intended to be used

Reply:
Just DRY principles - isn't it more efficient to assign this to a variable that to call the same function repeatedly? I am happy to be corrected, but am interested to understand why assets are not meant to be used this way.